src/cloudRun.ts

```diff
@@ -311,7 +311,7 @@ export class CloudRun {
    */
   async pollService(serviceResponse: run_v1.Schema$Service): Promise<string> {
     let url = getUrl(serviceResponse);
-    const maxAttempts = 60; // Timeout after 300 seconds
+    const maxAttempts = 120; // Timeout after 300 seconds
```

Comment:
I found during deletion I needed more time BUT I'm not sure if that was a fluke... I can bump down or update the seconds*